src/styles/calendar.scss

```
@@ -0,0 +1,22 @@
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: calculateCalendarWidth($calendarDayBoxSize, $calendarPadding, $gap);
+  padding: $calendarPadding;
+
+  &__day {
+    @extend %flexCenter;
+    @include square($calendarDayBoxSize);
+    background-color: $calendarDayColor;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition: 0.5s;
```

indicate which property you use transition for 
```suggestion
    transition: transform 0.5s, background-color 0.5s;
```